### PR TITLE
[DevOps] adding readme for how to deploy elasticstack using helm charts

### PR DESCRIPTION
**Pull request by fxdgear**
# Link to issue

- Related: https://github.com/CodeForPhilly/chime/issues/3

# Changelog entry

- README on how to use helm to install elastic stack into an elastic-stack namespace in a k8s cluster. 

# Additional info

This has been deployed already for testing into the `test-es` namespace on preprod. This namespace can be deleted at anytime. 

I'd suggest a review of the available overrides for each helmchart to ensure that the right settings are getting set that meet the needs for this project. 

The default settings have been applied to each service in the Elastic Stack so far and that means:

* metricbeat is pulling metric data from all pods in all namespaces
* filebeat is pulling all logs from all pods in all namespaces
* kibana is running without an ingress, which means you need to configure port forwarding to access kibana
* elasticsearch has no default username/password (also there is no ingress so you must have port forwarding enabled to access elasticsearch). 
* AMP server is configured to connect to elasticsearch, but no data is being shipped to APM server yet. This is something that the CHIME app would need to be updated to support. 

---

### <center>For Reviewer</center>

- [ ] Submitted under the correct tag
- [ ] Runs without error or warning
- [ ] Documentation  revised / included
- [ ] Unit tests included
- [ ] Issues referenced
- [ ] Does not create new bugs / issues

### CODE REVIEW

**Comment by MooseQuest** [comment]
For APM an issue has been opened to track #566

File: helm/values/elasticsearch.yaml [hunk 1]
```diff
@@ -0,0 +1,3 @@
+---
+
+clusterName: "chime-dev"
```
Comment (MooseQuest) [inline helm/values/elasticsearch.yaml]: The official cluster name I think is chime-bus, (it's back up in case prod get hits by a bus...)

**Review by MooseQuest — APPROVED** [review]
This looks good to me. This is enough of a readme to get started.

File: helm/values/elasticsearch.yaml (hunk 1 above)
Comment (fxdgear) [reply helm/values/elasticsearch.yaml]: ok Great. I'll make a change.

Comment (fxdgear) [reply helm/values/elasticsearch.yaml]:
```suggestion
clusterName: "chime-bus"
```

**Review by lottspot — CHANGES_REQUESTED** [review]
I think it's more appropriate if we file this information under `docs/`. Maybe `docs/helm` or something like that.

**Comment by lottspot** [comment]
Er... make that `docs/operations/helm`

**Comment by fxdgear** [comment]
> I think it's more appropriate if we file this information under `docs/`. Maybe `docs/helm` or something like that.

Moving to Docs makes sense. 

I was hoping that this README would live next to the customized values yaml files. 

BUT maybe this does make more sense in the docs directory and a more terse readme can live here, and reference the docs directory.

**Comment by lottspot** [comment]
@fxdgear We have generally been keeping documentation in `docs/operations/` and operational files under `k8s/`

**Comment by fxdgear** [comment]
@lottspot PTAL. I've updated the locations of files as well as updated the README to reference the new locations.

**Review by lottspot — APPROVED** [review]
Looks good!

**Pull request merged**